Source/Extensions/Blazorise.DataGrid/DataGrid.razor

```diff
@@ -247,8 +275,60 @@
         }
     };
 
+    protected RenderFragment groupFragment( (int iteration, IEnumerable<GroupContext<TItem>> groupedData) x ) => __builder =>
+    {
+        @foreach ( var group in x.groupedData )
+        {
+            <TableRowGroup @key="group.Key" Expanded="@group.Expanded" ExpandedChanged="@group.SetExpanded" IndentTableCells="x.iteration">

```

Comment:
Do we want to show one empty cell when we have only one group? `IndentTableCells="x.iteration + 1"`

Reply:
When there's only one group. There are no empty cells at all.
So the number of indented cells is x — 1 where x is the number of groups.

Would you think it looks better to do an indent cell per group in the header and row content but on the group row it is still x - 1?

Reply:
From my investigation, some are indenting the first group; some are not. We can leave it as this for now. The first group without indent.